package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ngl",
+  "name": "@ccdc-opensource/ngl",
   "version": "2.0.0-dev.39",
```

Comment:
What do we do about our version of the code - do we increase the version number or not?

Reply:
The version here will not change but on the build it will increment it i.e 2.0.0-dev.40 now we could change this from dev to ccdc for example